tools/postman/scripts/transform-for-api.sh

```diff
@@ -50,7 +50,7 @@ sed 's/\\"value\\": \\"<Circular reference to #[^>"]* detected>\\"//g' intermedi
 
 echo "Updating name with version $current_api_revision"
 jq --arg api_version "$current_api_revision" \
-  '.collection.info.name = ("MongoDB Atlas Administration API " + $api_version)' \
+  '.collection.info.name = ("⭐MongoDB Atlas Administration API " + $api_version)' \
```

Comment:
```suggestion
  '.collection.info.name = ("[Latest] MongoDB Atlas Administration API " + $api_version)' \
```
I am wondering why we went for the ⭐  . Isn't clearer to use Latest instead?

Reply:
The [discussion](https://mongodb.slack.com/archives/C07D6N6ECQP/p1733738821405619) started with using ⭐. I'm fine with changing it to [Latest]. Has there been a product decision on this? cc @matt-condon

Reply:
@yelizhenden-mdb Go ahead and use latest as star might not be universally recognizable across the globe as "first"/"latest".